docs/conf.py

```diff
@@ -239,7 +239,10 @@ def rstjinja(app, docname, source):
 # "version" configuration parameter
 edit_on_github_project = "sunpy/sunpy"
 if 'dev' not in release:
-    edit_on_github_branch = f"{version}"
+	edit_on_github_branch = f"{version}"
+	if version[0]=='0':
```

Comment:
Could you add a comment in the code above this line to say what it is doing? I'm a bit confused.